src/js/index.js

```diff
@@ -1,5 +1,8 @@
 'use strict';
 
+// include all our style information
+require('../scss/main.scss');
```

Comment:
@lirsacc Also for now I just require all our SCSS upon launch. We could start making this more granular down the line, although as we move to React I imagine we will just start using more inline styles.